### PR TITLE
Update edge install script text to account for viz

### PR DESCRIPTION
**Pull request by alpeb**
Fixes linkerd/linkerd2#5773

```console
$ LINKERD2_VERSION=stable-2.9.3 run.linkerd.io/public/install

Validating checksum...
Checksum valid.

Linkerd stable-2.9.3 was already downloaded; making it the default 🎉

To force re-downloading, delete '/home/alpeb/.linkerd2/bin/linkerd-stable-2.9.3' then run me again.

Add the linkerd CLI to your path with:

  export PATH=$PATH:/home/alpeb/.linkerd2/bin

Now run:

  linkerd check --pre                     # validate that Linkerd can be installed
  linkerd install | kubectl apply -f -    # install the control plane into the 'linkerd' namespace
  linkerd check                           # validate everything worked!

You can also obtain observability features by installing the viz extension:

  linkerd viz install | kubectl apply -f -  # install the viz extension into the 'linkerd-viz' namespace
  linkerd viz check                         # validate the extension works!
  linkerd dashboard                         # launch the dashboard

Looking for more? Visit https://linkerd.io/2/next-steps
```

### CODE REVIEW

File: run.linkerd.io/public/install [hunk 1]
```diff
@@ -16,7 +16,12 @@ happyexit() {
   echo "  linkerd check --pre                     # validate that Linkerd can be installed"
   echo "  linkerd install | kubectl apply -f -    # install the control plane into the 'linkerd' namespace"
   echo "  linkerd check                           # validate everything worked!"
-  echo "  linkerd dashboard                       # launch the dashboard"
+  echo ""
+  echo "You can also obtain observability features by installing the viz extension:"
+  echo ""
+  echo "  linkerd viz install | kubectl apply -f -  # install the viz extension into the 'linkerd-viz' namespace"
+  echo "  linkerd viz check                         # validate the extension works!"
+  echo "  linkerd dashboard                         # launch the dashboard"
```
Comment (adleong) [inline run.linkerd.io/public/install:24]: shouldn't this be `linkerd viz dashboard`?

Comment (alpeb) [reply run.linkerd.io/public/install:24]: Indeed!

**Comment by adleong** [comment]
Should we maybe merge only the `install-edge` change here, and separately  merge the `install` change into the `release/2.10` branch?

**Comment by alpeb** [comment]
> Should we maybe merge only the `install-edge` change here, and separately merge the `install` change into the `release/2.10` branch?

Yep, makes sense. I amended the current PR and will issue an new one shortly for `install`

**Review by adleong — APPROVED** [review]

**Pull request merged**